charts/geonode/values.yaml

```diff
@@ -348,9 +348,9 @@ geoserver:
     container_name: geoserver-wait-for-db
     image:
       # init container image
-      name: jwilder/dockerize
+      name: governmentpaas/psql
```

Comment:
can we use an official image instead?

Reply:
use the big image

Reply:
evantually the same used by the zalando cluster if available in the values file

Reply:
Ok. My latest commit uses `postgres:16-alpine`.